internal/pkg/v2/infrastructure/redis/device_service.go

```diff
@@ -18,22 +18,49 @@ import (
 
 const DeviceServiceCollection = "v2:deviceService"
 
+// deviceServiceStoredKey return the device service's stored key which combines the collection name and object id
+func deviceServiceStoredKey(id string) string {
+	return fmt.Sprintf("%s:%s", DeviceServiceCollection, id)
+}
+
+// deviceServiceExistById checks whether the device service exists by id
+func deviceServiceExistById(conn redis.Conn, id string) (bool, errors.EdgeX) {
+	exists, err := redis.Bool(conn.Do(EXISTS, deviceServiceStoredKey(id)))
+	if err != nil {
+		return false, errors.NewCommonEdgeX(errors.KindDatabaseError, "device service existence check by id failed", err)
+	} else if exists {
+		return true, nil
+	}
+	return false, nil
```

Comment:
replace these lines with `return exists, nil`

Reply:
Modified.

Reply:
@weichou1229 , Not seeing the changes. Did they get pushed successfully?

Reply:
Sorry, pushed it again.